### ISSUE TITLE
False positive for io.dropwizard.logback:logback-throttling-appender:1.1.0

### ISSUE THREAD

**Issue by mtraynham** [dependency-check/DependencyCheck#2473]
**Describe the bug**
Seems to be a false positive reported for 
```
io.dropwizard.logback:logback-throttling-appender:1.1.0
```

```
logback-throttling-appender-1.1.0.jar (pkg:maven/io.dropwizard.logback/logback-throttling-appender@1.1.0, cpe:2.3:a:logback:logback:1.1.0:*:*:*:*:*:*:*) : CVE-2017-5929
```

https://nvd.nist.gov/vuln/detail/CVE-2017-5929

My guess is the string `logback:logback` is matching and the version is less than the CVE.

**Version of dependency-check used**
5.3.0

**Log file**
```
> Task :dependencyCheckAggregate
Verifying dependencies for project foobar
Checking for updates and analyzing dependencies for vulnerabilities
----------------------------------------------------
.NET Assembly Analyzer could not be initialized and at least one 'exe' or 'dll' was scanned. The 'dotnet' executable could not be found on the path; either disable the Assembly Analyzer or configure the path dotnet core.
----------------------------------------------------
Generating report for project foobar
Found 1 vulnerabilities in project foobar

One or more dependencies were identified with known vulnerabilities in foobar:

logback-throttling-appender-1.1.0.jar (pkg:maven/io.dropwizard.logback/logback-throttling-appender@1.1.0, cpe:2.3:a:logback:logback:1.1.0:*:*:*:*:*:*:*) : CVE-2017-5929

See the dependency-check report for more details.

> Task :dependencyCheckAggregate FAILED
```

**To Reproduce**
1. Add Dropwizard 2 Logging as a dependency, `io.dropwizard:dropwizard-logging:2.0.1`
2. Run check

**Expected behavior**
Does not warn.

**Additional context**

**Comment by accwebs** [comment]
I see https://github.com/jeremylong/DependencyCheck/issues/2489 marked as fixed in 5.3.1 but I still seem to be getting the false positive in 5.3.1's output.  Am I just being dumb and doing something wrong or is this still pending a fix?

Thanks for all that you do :-)

**Comment by aikebah** [comment]
@accwebs When I run 5.3.1 on a project with io.dropwizard:dropwizard-logging:2.0.1 as a dependency there is no vulnerability reported for logback-throttling-appender

@jeremylong This one is a duplicate of #2489 and can be closed
